### PR TITLE
Use default query structure for network.js

### PR DESCRIPTION
**Pull request by CHelfgott**
We shouldn't need to make code changes every time we bring an organization online.  Use a default query structure so that any new organization pulls from our own tables unless we specifically override the logic for that organization (read Tula Salud).

### CODE REVIEW

File: routes/api/network.js [hunk 1]
```diff
@@ -6,7 +6,10 @@ const router = express.Router();
 
 router.get('/', authenticateResource('network'), (req, res) => {
   const { user } = req;
-  const Query = new q[user.organizationName].Network(user);
+  var Query = new q['Default'].Network(user);
+  if q.hasOwnProperty(user.organizationName) {
```
Comment (klandell) [inline routes/api/network.js]: Syntax error here...  But this whole block can be structured differently.

```
const Query = new (q[user.organizationName] || q.Default).Network(user);
```

File: routes/api/queries/index.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 const Gambia = require('./gambia');
 const LastMile = require('./lastmile');
 const Tula = require('./tula');
+const Query = require('./query');
 
 module.exports = {
   Gambia,
```
Comment (klandell) [inline routes/api/queries/index.js]: You should be able to delete the Gambia and Last mile specific queries and their references here.  The only organization that has anything special is Tula Salid (if I remember correctly)

File: routes/api/queries/index.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
   'Last Mile Health': LastMile,
   'Tula Salud': Tula,
+  'Default': Query,
```
Comment (klandell) [inline routes/api/queries/index.js]: Oh, also, no need for quotes on Default here.

Comment (CHelfgott) [reply routes/api/queries/index.js]: done

File: routes/api/queries/index.js (hunk 2 above)
Comment (CHelfgott) [reply routes/api/queries/index.js]: done

File: routes/api/network.js (hunk 1 above)
Comment (CHelfgott) [reply routes/api/network.js]: done

File: routes/api/network.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ const router = express.Router();
 
 router.get('/', authenticateResource('network'), (req, res) => {
   const { user } = req;
-  const Query = new q[user.organizationName].Network(user);
+  const Query = new (q[user.organizationName] || q['Default']).Network(user);
```
Comment (klandell) [inline routes/api/network.js]: Please use dot notation

Comment (CHelfgott) [reply routes/api/network.js]: done

**Comment by klandell** [comment]
@CHelfgott I changed the base branch to be the new-ui branch (which will be going into staging soonish).  Can you please fix the merge conflict with this branch?  I added coded to set the root name of the Network. 
 We should definitely use the fixtures table instead to hold the name of the root node.  Can you please add the root node to the fixtures table and make any required changes to get it to work?

**Pull request merged**